README.md

```diff
@@ -0,0 +1,25 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Abdulrahman's ؛ortfolio
+
+Simple Portfolio website
+
+![screenshot](images/my-portfolio.png)
+
+Simple Portfolio
+
+## Built With
+
+- HTML
+- CSS
+
```

Comment:
- [ ] [OPTIONAL] You can add the following section to help users, especially those not so familiar with coding to set up and view your work 
```
### Setup
To get a local copy up and running follow these simple example steps.
- Clone the repository to your computer using git clone [Git Clone Url](https://github.com/abdulrahmanshr75/Microverse-Portfolio.git)
- Cd into the folder
- Open index.html with live server
- Or you can Fork it instead, and create a pull request to suggest some changes
```